### PR TITLE
Html manual, (bowtie for event)

### PR DESCRIPTION
**Pull request by maranget**
This PR fixes small problems in the html manual.

In particular it solves issue #10287.

### CODE REVIEW

**Review by gasche — APPROVED** [review]
This looks fine and I liked the PR title "bowtie for event", we don't have enough mysteries in OCaml development! I have no opinion on the quote->maintitle formatting change, I think @Octachron should have a look to that one.

**Comment by maranget** [comment]
> I have no opinion on the quote->maintitle formatting change, I think @Octachron should have a look to that one.

The `\rule{}{}` command was producing no output and a warning. I have [changed](https://github.com/ocaml/ocaml/pull/10288/files#diff-30ccdd13dbdfc5f39d09305b86b5ebac5efd401f1b5ddde206a5fc53843c7322) the formatting commands and visually checked that the rendering was satisfying.

File: manual/src/allfiles.etex [hunk 1]
```diff
@@ -23,17 +23,16 @@
...
 This manual is also available in
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.pdf}{PDF},
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.txt}{plain text},
 as a
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman-html.tar.gz}{bundle of HTML files},
 and as a
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.info.tar.gz}{bundle of Emacs Info files}.
-\rule{}{}
-\end{quote}
+\end{maintitle}
```
Comment (Octachron) [inline manual/src/allfiles.etex:35]: This is a hevea 2.35 change, isn't it?

Comment (maranget) [reply manual/src/allfiles.etex:35]: It is, before we had horizontal rules and no warning. Losing the rule does not look to be such a problem

Comment (Octachron) [reply manual/src/allfiles.etex:35]: The maintitle rendering does seem fine to me. In term of structure, having two `div.maintitle`s seems not completely ideal. However, we can improve (or not) that point later and the postprocessed manual is unaffected.

**Comment by gasche** [comment]
If I understand correctly, @Octachron is satisfied and this is all good for merging. (Waiting until the next tick to click the button.)

**Pull request merged**